### PR TITLE
Hook up disqus

### PR DESCRIPTION
**Pull request by LukeSettle**
Hey. I did it. Give me my money.

### CODE REVIEW

File: source/layouts/blog.erb [hunk 1]
```diff
@@ -3,3 +3,16 @@
   <time><%= current_article.date.strftime("%B %d %Y") %></time>
   <%= yield %>
 <% end %>
+<div id="disqus_thread"></div>
```
Comment (kurtisnelson) [inline source/layouts/blog.erb:36]: I think this needs to be inside of the layout.

**Comment by kurtisnelson** [comment]
Closes #2

File: source/layouts/layout.erb [hunk 2]
```diff
@@ -34,6 +34,6 @@
     </nav>
     <div class="container">
       <%= yield %>
-    </div><!-- /.container -->
+    </div><!-- /.container -->
```
Comment (kurtisnelson) [inline source/layouts/layout.erb:73]: you added a trailing space

**Comment by BWheatie** [comment]
This needs to be merged but I am unable to do it.

**Comment by kurtisnelson** [comment]
Luke, you need to run `git rebase master` while this branch is checked out.

**Comment by kurtisnelson** [comment]
:sob:

**Pull request merged**